iocage/cli/fetch.py

```diff
@@ -45,51 +48,94 @@ def validate_count(ctx, param, value):
     else:
         return int(value)
 
+def release_prompt():
+    i = 0
+    default = None
+    for available_release in host.distribution.releases:
+        if available_release.name == host.release_version:
+            default = i
+            print(f"[{i}] \033[1m{available_release.name}\033[0m")
```

Comment:
this seems to be something that's better wrapped by a library that understands whether it's talking to an interactive tty, or being piped into a file